dhis-2/dhis-api/src/main/java/org/hisp/dhis/dataset/Section.java

```diff
@@ -67,6 +67,8 @@
 
     private boolean showColumnTotals;
 
+    private boolean disableDataElementAutoGroup;
```

Comment:
Since this is a primitive boolean, we need to add a Flyway migration which sets this value to false for all existing sections. If not we will get NPEs for existing databases.

Reply:
yes, there is a migration script